contracts/interfaces/IOrderMixin.sol

```diff
@@ -172,11 +175,12 @@ interface IOrderMixin {
     ) external returns(uint256 makingAmount, uint256 takingAmount, bytes32 orderHash);
 
     /**
-     * @notice Same as `fillContractOrder` but calls permit first.
+     * @notice Same as `fillContractOrder` but allows to specify arguments that are used by the taker.
```

Comment:
It might make sense to provide more details about these parameters and their usage; idk, what do you think?

Reply:
Related to that is the description for `args` parameter:
``` solidity
* @param args Arguments that are used by the taker (target, extension, interaction, permit)
```
Do you think we should also add more details to the "notice" part?

Reply:
It is not clear why do we need to pass certain parameters separately in the method when they are already contained within one of the provided parameters?

Reply:
Having looked into it further, everything now seems clear to me